README.md

```diff
@@ -1,5 +1,9 @@
+:warning: This repository contains a standalone example to be used as a reference to help our partners integrate with Tilled. It is **not** intended be implemented in a production environment nor is it intended to be installed as a dependency in any way.
```

Comment:
I think the word `to` is missing after `...intended`. Other than that, looks great!